packages/mui-joy/src/styles/defaultTheme.ts

```diff
@@ -68,15 +73,18 @@ export const getThemeWithVars = (
       solidHover: createVariant('solidHover', defaultTheme),
       solidActive: createVariant('solidActive', defaultTheme),
       solidDisabled: createVariant('solidDisabled', defaultTheme),
-
-      // variant overrides
-      plainOverrides: createTextOverrides(defaultTheme),
-      outlinedOverrides: createTextOverrides(defaultTheme),
-      softOverrides: createTextOverrides(defaultTheme),
-      solidOverrides: createContainedOverrides(defaultTheme),
     },
     defaultTheme.variants,
   );
+  defaultTheme.variantOverrides = deepmerge(
+    {
+      plain: createPlainOverride(defaultTheme),
+      outlined: createPlainOverride(defaultTheme),
```

Comment:
```suggestion
      outlined: createOutlinedOverride(defaultTheme),
```
?

Reply:
I think I understand now why it is the same (these two variants, have no overrides basically :))